paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -243,6 +256,9 @@ class PaymentSheet internal constructor(
             externalPaymentMethodConfirmHandler?.let {
                 ExternalPaymentMethodInterceptor.externalPaymentMethodConfirmHandler = it
             }
+            cvcRecollectionEnabledCallback?.let {
+                CvcRecollectionCallbackHandler.isCvcRecollectionEnabledCallback = it
```

Comment:
We need to clear this. See https://github.com/stripe/stripe-android/blob/a35d3b5dad49b01624c7e9b402ea7b794592b0cd/paymentsheet/src/main/java/com/stripe/android/paymentsheet/DefaultPaymentSheetLauncher.kt#L30

Reply:
Done 👍

Reply:
Also need to clear it for flow controller: https://github.com/stripe/stripe-android/blob/ff4b0099ab935236d738aa41784d4d0afe2811d1/paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt#L226

Reply:
Done 👍